gradio/components.py

```diff
@@ -210,6 +211,10 @@ def get_load_fn_and_initial_value(value):
             load_fn = None
         return load_fn, initial_value
 
+    @abstractmethod
+    def as_example(self, input_data):
```

Comment:
Marking this as an abstract method so we don't forget to add it for new components. The trade-off is that there will be some repeated 

```python
    def as_example(self, input_data):
        return input_data
```

code but I think it's worth it for the "safety" of knowing a someone had to type out an implementation as opposed to just forgetting and inheriting from IO component.